sematic/ui/packages/common/src/pages/RunSearch/RunList.tsx

```diff
@@ -141,55 +133,45 @@ const columns = [
             }
         },
         header: 'Status',
-        cell: info => <RunStatusColumn {...info.getValue()} />,
-    }),
-    columnHelper.display({
-        meta: {
-            columnStyles: {
-                width: "2.46792%",
-                minWidth: "40px"
-            }
-        },
-        header: '',
-        id: 'goto',
-        cell: _ => <ChevronRight style={{cursor: 'pointer'}}/>,
-    }),
+        cell: info => <RunStatusColumn {...info.getValue() as any} />,
+    })
 ]
 
-const data: Array<typeof RunData> = [
-    RunData,
-    RunData,
-    { ...RunData, resolvedAt: '2021-10-01T12:10:00Z', status: 'SUCCESS' },
-    { ...RunData, failedAt: '2021-10-01T12:10:00Z', status: 'FAILED' },
-    { ...RunData, canceledAt: '2021-10-01T12:10:00Z', status: 'CANCELLED' },
-    { ...RunData, status: 'SCHEDULED' },
-    RunData,
-    RunData,
-    RunData,
-    RunData,
-    { ...RunData, canceledAt: '2021-10-01T12:10:00Z', status: 'CANCELLED' },
-    { ...RunData, status: 'SCHEDULED' },
-    RunData,
-    RunData,
-    { ...RunData, canceledAt: '2021-10-01T12:10:00Z', status: 'CANCELLED' },
-]
+interface RunListProps {
+}
+
+const RunList = (props: RunListProps) => {
+    const { runs, page, isLoading, totalPages, totalRuns, nextPage, previousPage } = useRunsPagination();
+
+    const { setIsLoading } = useContext(LayoutServiceContext);
 
-const RunList = () => {
     const tableInstance = useReactTable({
-        data,
+        data: runs,
```

Comment:
```suggestion
        runs: runs,
```

Reply:
This is the formal parameter name of `useReactTable`, which is an external dependency. unfortunately, it cannot be changed.